pom.xml

```diff
@@ -314,7 +342,9 @@
             <groupId>com.googlecode.json-simple</groupId>
             <artifactId>json-simple</artifactId>
         </dependency>
-
+		
+		
+		
```

Comment:
Could you please remove these extra lines?

Reply:
removed and updated in my repository